service/src/main/java/com/ge/predix/acs/attribute/connector/management/AttributeConnectorServiceImpl.java

```diff
@@ -46,18 +42,25 @@ public boolean upsertResourceConnector(final AttributeConnector connector) {
     @Override
     public AttributeConnector retrieveResourceConnector() {
         ZoneEntity zoneEntity = this.zoneResolver.getZoneEntityOrFail();
-        return this.connectorConverter.toConnector(zoneEntity.getResourceAttributeConnector());
+        try {
```

Comment:
My understanding is that is the current format of the Json. The Object Mapper deserializes the object into those fields.